tests/pyrog/api/test_attribute.py

```diff
@@ -0,0 +1,76 @@
+import pytest
+from faker import Faker
+
+from django.urls import reverse
+
+faker = Faker()
+
+pytestmark = pytest.mark.django_db
+
+
+@pytest.mark.parametrize(
+    "path, slice_name, definition_id, status_code",
+    [
+        (faker.word(), faker.word(), faker.word(), 201),
+    ],
+)
+def test_create_attribute(
+    api_client,
```

Comment:
@clementjumel could we use the same idea with a fixture?
like here:
https://github.com/arkhn/fhir-river/blob/50634ae078a44039d01d082f8d81ce2b9e661a2e/tests/conftest.py#L16

Reply:
I guess so!
I opened an issue: https://github.com/arkhn/pypa/issues/161